.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -5,13 +5,13 @@ on:
   push:
     branches: [ "master" ]
     paths:
-      - "aiosmtpd/*"
+      - "aiosmtpd/**"
       - "setup.cfg"  # To monitor changes in dependencies
   # This is for PRs
   pull_request:
     branches: [ "master" ]
     paths:
-      - "aiosmtpd/*"
+      - "aiosmtpd/**"
```

Comment:
Enlarge triggering event scope. `*` triggers only for files within `aiosmtpd`, `**` is recursive.